system/keyboard.c

```diff
@@ -255,5 +258,13 @@ char get_key(void)
         }
     }
 
+    if (enable_tty) {
+        uint8_t c = tty_get_key();
+        if (c != 0xFF) {
+            if(c == 0x0D) c = '\n'; // Enter
```

Comment:
Whitespace.

Reply:
I'm doing a pass to solve all tabs->spaces, but I can't see the issue here.

Reply:
Most other places use `if (` :)

Reply:
ah yes, I was looking for superfluous spaces :) 

I always had trouble switching between coding convention. Ie: many devs I work with consider anything other than a 2-spaces indent as heretic (but they also loves to write obscure one liner...) .

Reply:
I'm quite used to switching. For my own code I use Camel case and an indentation of 2. I also help maintain Icarus Verilog - the only code base I know of that uses an indentation of 6.